website/routes.py

```diff
@@ -1596,6 +1597,53 @@ def make_url_map(app):
             json_renderer,
         ),
 
+        Rule(
+            [
+                '/project/<pid>/mailing_list/',
```

Comment:
Standard confirmation question: have all new routes been confirmed by the routing tsars?